src/release/breaking-changes/3-13-deprecations.md

```diff
@@ -0,0 +1,456 @@
+[`MaterialScrollBehavior`]: {{site.api}}/flutter/material/MaterialScrollBehavior-class.html
+[`ThemeData`]: {{site.api}}/flutter/material/ThemeData-class.html
+[`GlowingOverscrollIndicator`]: {{site.api}}/flutter/widgets/GlowingOverscrollIndicator-class.html
+[`StretchingOverscrollIndicator`]: {{site.api}}/flutter/widgets/StretchingOverscrollIndicator-class.html
+
+[#100234]: {{site.repo.flutter}}/pull/100234
+[#133181]: {{site.repo.flutter}}/pull/133181
+
+---
+
+### Updates to ImageProvider and PaintingBinding
+
+Package: flutter
+Supported by Flutter Fix: no
+
+The `instantiateImageCodec` method of `PaintingBinding`, as well as the `load`
+method of `ImageProvider` and the associated `DecoderCallback` were all
+deprecated in v2.13.
+
+The replacement respective replacements are
+`PaintingBinding.instantiateImageCodecFromBuffer`, `ImageProvider.loadBuffer`
+and `DecoderBufferCallback`.
+
+This change enabled faster performance in image loading by using a buffer.
+
+**Migration guide**
+
+Code before migration:
+
+```dart
+PaintingBinding.instance.instantiateImageCodec
+```
+
+Code after migration:
+
+```dart
+PaintingBinding.instance.instantiateImageCodecFromBuffer
+```
+
+**References**
+
+API documentation:
+
+* [`PaintingBinding`][]
+* [`ImageProvider`][]
+* [`DecoderBufferCallback`][]
+
+Relevant PRs:
+
+* Deprecated in [#103496][]
+* Removed in [#132679][]
+
+[`PaintingBinding`]: {{site.api}}/flutter/painting/PaintingBinding-mixin.html
+[`ImageProvider`]: {{site.api}}/flutter/painting/ImageProvider-class.html
+[`DecoderBufferCallback`]: {{site.api}}/flutter/painting/DecoderBufferCallback.html
+
+[#103496]: {{site.repo.flutter}}/pull/103496
+[#132679]: {{site.repo.flutter}}/pull/132679
+
+---
+
+### TestWindow properties
+
+Package: flutter_test
+Supported by Flutter Fix: no
+
+In preparation for multi-window support, many deprecated properties from
+`TestWindow`, which itself is entirely deprecated, has been removed. While the
+deprecation timeframe on `TestWindow` has not yet elapsed, migrating the
+expired properties this cycle will be excellent preparation for fully migrating
+off of `TestWindow`.
+
+The following properties were removed:
+
+* `localeTestValue`
+* `clearLocaleTestValue`
+* `localesTestValue`
+* `clearLocalesTestValue`
+* `initialLifecycleStateTestValue`
+* `textScaleFactorTestValue`
+* `clearTextScaleFactorTestValue`
+* `platformBrightnessTestValue`
+* `clearPlatformBrightnessTestValue`
+* `alwaysUse24HourFormatTestValue`
+* `clearAlwaysUse24HourTestValue`
+* `brieflyShowPasswordTestValue`
+* `defaultRouteNameTestValue`
+* `clearDefaultRouteNameTestValue`
+* `semanticsEnabledTestValue`
+* `clearSemanticsEnabledTestValue`
+* `accessibilityFeaturesTestValue`
+* `clearAccessibilityFeaturesTestValue`
+
+This update to `TestWindow` is covered more extensively in the
+[in-depth migration guide][].
+
+[in-depth migration guide is also available]: {{site.url}}/release/breaking-changes/window-singleton
```

Comment:
**suggestion (non-blocking):** Consider a more concise phrasing. (Also update the link to match.)
```suggestion
To learn more about this `TestWindow` update, check out
[`TestWindow` migration guide][].

[`TestWindow` migration guide]: {{site.url}}/release/breaking-changes/window-singleton
```